Source/AFError.swift

```diff
@@ -466,6 +552,14 @@ extension AFError: LocalizedError {
             return "Server trust evaluation failed."
         case let .urlRequestValidationFailed(reason):
             return "URLRequest validation failed due to reason: \(reason.localizedDescription)"
+        case let .createUploadableFailed(error):
+            return "Creating uploadable failed with error: \(error.localizedDescription)"
```

Comment:
We prefer these to read "Thing creation failed with error".

Reply:
Edited.